docs/salesforce/release-notes.mdx

```diff
@@ -21,9 +21,21 @@ import {
 
 # Release Notes
 
+## 3.9
+
+_March 12, 2024_
+
+** Release Notes **
+
+- Agents will now be able to set text direction or direction will be detected automatically.
```

Comment:
suggestion - 
Text direction in messages will be detected automatically, and agents can set the text direction manually.